contracts/script/UpgradeLightClient.s.sol

```diff
@@ -0,0 +1,38 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.19;
+
+import { Script } from "forge-std/Script.sol";
+
+import { LightClientV2 as LCV2 } from "../src/LightClientV2.sol";
+import { LightClient as LC } from "../src/LightClient.sol";
+
+contract UpgradeLightClientScript is Script {
+    /// @notice runs the upgrade
+    /// @param mostRecentlyDeployedProxy address of deployed proxy
+    /// @return address of the proxy
+    /// TODO get the most recent deployment from the devops tooling
+
+    function run(address admin, address mostRecentlyDeployedProxy) external returns (address) {
+        LCV2 newLC = new LCV2();
+        address proxy = upgradeLightClient(admin, mostRecentlyDeployedProxy, address(newLC));
```

Comment:
why separate out as a function, it'll just be extra `JUMP` without much readability benefit here?

Reply:
Agreed. Change done in 6c8e4cd.